h/static/styles/annotations.scss

```diff
@@ -63,12 +63,62 @@
 
 .annotation-quote {
   @include quote;
+  height: 100%;
+  max-height: 2.9em;
+  overflow: hidden;
+
+  &.show-full-excerpt {
+    max-height: 100%;
+  }
   del {
     background:#ffe6e6;
   }
   ins {
     background:#e6ffe6;
   }
+
+  .more {
+    position: relative;
+    top: 3.35em;
+    float: right;
+    margin-top: -2em;
+
+    &:before {
+      opacity: .9;
+      content: " ";
+      margin-left: -3em;
+      padding-right: 40px;
+      background: -webkit-gradient(linear, left top, right top, from(rgba(255, 255, 255, 0)), to(white), color-stop(50%, white));
+      background: -moz-linear-gradient(to right, rgba(255, 255, 255, 0), white 50%, white);
+      background: -o-linear-gradient(to right, rgba(255, 255, 255, 0), white 50%, white);
+      background: -ms-linear-gradient(to right, rgba(255, 255, 255, 0), white 50%, white);
+      background: linear-gradient(to right, rgba(255, 255, 255, 0), white 50%, white);
+    }
+  }
+}
+
+.annotation-body {
+  max-height: 14.8em;
+  overflow: hidden;
+  position: relative;
+
+  &.show-full-excerpt {
+    max-height: 100%;
+  }
+
+  .more {
+    top: 12em;
+    padding-top: 1.5em;
```

Comment:
It's strange to me to have this padding if it's positioned absolutely. If you want to use the padding to make the click target easier, make it symmetrical. If you're using it for positioning, just use positioning.